### PR TITLE
Model

### PR DESCRIPTION
**Pull request by ishitapadhiar**
Revised model file, added the User as a separate model, so it can have its own database and functions.

### CODE REVIEW

**Review by ominhas — COMMENTED** [review]
Good work in general.

A few things:
- word (under Tag), rcs_id (under User) should all be set as primary keys. This prevents Django from creating keys of its own.
- Is the "expected_sig" field necessary? If I'm correct, this should be the same for every petition, so there shouldn't be a reason to store it in every petition.
- It might be good to limit the "Description" field to 4000 characters (like in the old site).
- Not sure if this is done already, but are the "tags" under Petitions linked to the "word" in the Tag table?
- Is "author" under Petitions linked to "rcs_id" under User?
- I don't think it's strictly necessary to add "signatures" under Petitions, but I think it's a good addition. This also needs to be linked to the id under  the Signature table.
- The Signature field should also mention the Petition that is signed, and link that to the petition id under Petitions.
- There's a typo in line 11, "petitions"

Also I could be wrong about things in my comments, so feel free to ignore me if I'm being dumb.

**Review by andrewdaoust — DISMISSED** [review]
See notes from @ominhas but also make sure you are still following Python styling guides: https://github.com/google/styleguide/blob/gh-pages/pyguide.md

**Comment by andrewdaoust** [comment]
Good additions in last commit @ishitapadhiar.  Keep building out the model with the other functions you left a code comment about.

File: index/models.py [hunk 1]
```diff
@@ -2,28 +2,108 @@
...
+	ID = models.IntegerField(999999, primary_key=True)
+	archived = models.BooleanField(default=False)
+				
+	created_date = models.DateTimeField(db_index=True, default=timezone.now)	#files the date created
 	expected_sig = models.IntegerField(300)		#the expected signature to move to the next step
 
-	tags = models.ManyToField(Tag, related_name='petititons')
-	signatures = models.ManyToField(Signature, related_name='petitions')
+	author = models.ForeignKey(User, related_name='petitions')	#the author of the petition
+	tags = models.ManyToManyField(Tag, related_name='petitions')	#the tags, probably a max of 3
```
Comment (garoller) [inline index/models.py:14]: Good change to `ManyToManyField`

File: index/models.py [hunk 2]
```diff
@@ -2,28 +2,108 @@
 
 class Petitions(models.Model):
```
Comment (garoller) [inline index/models.py]: Should we change it to `Petition` instead of `Petitions`?

File: index/models.py [hunk 3]
```diff
@@ -2,28 +2,108 @@
...
+
+	def get_name(self):
+		return self.name
+
+	def get_status(self):
+		if self.admin
+			return True
+
+class Response(models.Model):
+	senator_invetigation = models.BooleanField(default=False)
```
Comment (garoller) [inline index/models.py]: I think this should be `senator_investigation`

File: index/models.py [hunk 4]
```diff
@@ -2,28 +2,108 @@
...
+	committee_formed = models.BooleanField(default=False)
+	vote_resolution = models.BooleanField(default=False)
+	vote_referendum = models.BooleanField(default=False)
+	refer_to_other = models.BooleanField(default=False)
+	
+	invetigation_info = models.CharField(max_length=1000)
```
Comment (garoller) [inline index/models.py]: Same as above `investigation_info`

File: index/test_petitions_model.py [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import unittest
+import calc
+
+class TestPetitionsModel(unittest.TestCase):
+
+	def test_
```
Comment (garoller) [inline index/test_petitions_model.py]: If you're able to add unit tests now for the functions that you added in `Petitions` and `User`, that would be great!

**Review by garoller — DISMISSED** [review]
There are just a few spelling things to fix and adding those unit tests. It looks good otherwise!

**Review by garoller — APPROVED** [review]
Everything looks good, thank you!

**Pull request merged**